cypress/support/constants/constants.js

```diff
@@ -200,6 +199,7 @@ module.exports = {
   LIFECYCLE_HISTORY_SCHEMA_PATH: 'schemas/lifecycleHistorySchema',
   LIFECYCLE_INTENT: 'Lifecycle intent sent to application: ',
   LIFECYCLE_NOTIFICATION_GENERATED: 'Lifecycle events generated Req #',
+  LIFECYCLE_NOTIFICATION_NOT_GENERATED: 'Lifecycle events not generated',
   LIFECYCLE_STATE: 'Lifecycle.state',
```

Comment:
why to create this ?

Reply:
check with nandana on the logs. It was properly fixed in on of the ticket with a single log https://ccp.sys.comcast.net/browse/FIRECERT-2066